tika-parsers/src/main/resources/org/apache/tika/parser/recognition/tf/InceptionVideoRestDockerfile

```diff
@@ -15,37 +15,34 @@
 # limitations under the License.
 #
 
-# Take base as Ubuntu: 16.04, then you can simply apt-get install ffmpeg
 FROM ubuntu:16.04
 MAINTAINER Thamme Gowda <thammegowda@apache.org> and Kranthi Kiran GV <kkranthi@student.nitw.ac.in> and \
            Madhav Sharan <msharan@usc.edu> and Chris Mattmann <mattmann@apache.org>
 
+# Install missing part of ubuntu core + python + building dependencies
 RUN apt-get update && \
-      apt-get install -y wget && \
-      apt-get install -y unzip && \
-      apt-get install -y curl
-
-
-# Install ffmpeg
-RUN apt-get install -y ffmpeg
-
-# Dependencies
-RUN apt-get install -y build-essential cmake pkg-config \
+      apt-get install -y wget git unzip curl \
+      libtcmalloc-minimal4 software-properties-common apt-utils \
+      build-essential cmake pkg-config \
       libjpeg8-dev libtiff5-dev libjasper-dev libpng12-dev \
       libavcodec-dev libavformat-dev libswscale-dev libv4l-dev \
       libxvidcore-dev libx264-dev \
       libgtk2.0-dev \
       libatlas-base-dev gfortran \
       python2.7-dev \
       python-pip
+
+# Install ffmpeg
+RUN apt-get install -y ffmpeg
+
 RUN \
       pip install --upgrade pip && \
       pip install numpy
 
+# Download OpenCV + OpenCV_Contrib
 WORKDIR /usr/local/src
 
-# Download OpenCV 3 + Contrib
-RUN \ 
```

Comment:
Thought this would make much sense if it's called OpenCV_Contrib rather than Contrib.